R/build_universe.R

```diff
@@ -13,10 +13,6 @@ build_universe <- function(input = getwd(), output = "packages.json") {
   assert_character_scalar(output, "invalid output")
   assert_file(input)
   packages <- list.files(input, all.files = FALSE, full.names = TRUE)
-  for (package in packages) {
-    result <- assert_package(package)
-    is.null(result) || stop(result, call. = FALSE)
-  }
```

Comment:
Do we need any guardrails here to prevent manually merged PRs causing errors in the following steps?

Reply:
Yes, you're right. Added and tested in https://github.com/wlandau/r.releases.utils/commit/42d3221bbfa6e4cfe66aa54a9ac989c3e7aa91d2.